generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -9,9 +9,19 @@
 import <%=packageName%>.security.AuthoritiesConstants;
 import <%=packageName%>.security.SecurityUtils;
 import <%=packageName%>.service.util.RandomUtil;
+<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>
+import <%=packageName%>.web.rest.util.PaginationUtil;
+<%_ } _%>
 import <%=packageName%>.web.rest.vm.ManagedUserVM;
```

Comment:
My last commit has been fixing this. I think it's a lot cleaner now: no class from the `web` layer is called in the service anymore (except the DTO).
I just mapped the page of user to a new one with the DTOs.